sftp-server.c

```diff
@@ -863,6 +866,12 @@ process_write(u_int32_t id)
 	    (r = sshbuf_get_string(iqueue, &data, &len)) != 0)
 		fatal_fr(r, "parse");
 
+#ifdef WINDOWS
+	if (add_mark_of_web(resolved_path_utf8(handle_to_name(handle))) == -1) {
```

Comment:
`resolved_path_utf8()` allocates memory for the resolved file path, correct?  So shouldn't we assign this to a local variable and the call `LocalFree()` on it?